### PR TITLE
feat(filter.dropdown): Add ability to filter a single selection dropd…

### PR DESCRIPTION
**Pull request by corinnekrych**
…ownbox

WIP - do not merge yet, early push to get feedback from @mitchharpur 
Fixes https://github.com/fabric8io/fabric8-ui/issues/1489

### CODE REVIEW

File: src/app/space-wizard/services/fabric8-app-generator.service.ts [hunk 1]
```diff
@@ -336,7 +336,8 @@ export class Fabric8AppGeneratorService extends AppGeneratorService {
         return FieldWidgetClassificationOptions.SingleInput;
       }
       case 'uiselectone': {
-        return FieldWidgetClassificationOptions.SingleSelectionDropdown;
+        return FieldWidgetClassificationOptions.FilteredDropdown;
+        //return FieldWidgetClassificationOptions.SingleSelectionDropdown;
```
Comment (corinnekrych) [inline src/app/space-wizard/services/fabric8-app-generator.service.ts]: @mitchharpur not sure how to hook to be ale to display the 2 different Dropdowns. We can't just be based on forge reponse. or do we want all dropdown to be filterable?

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1502-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/space-wizard/components/app-generator-single-selection-dropdown/app-generator-filtered-dropdown.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+              <div class="item-name">{{choice.name}}</div>
+              <div *ngIf="choice.description" class="item-description">{{choice.description||''}}</div>
+            </div>
+          </div>
+        </div>
+      </li>
+    </ul>
+    <span *ngIf="field.display.enabled" class="input-group-addon" (click)="field.display.open =!field.display.open">
+      <span class="caret"></span>
+      <span class="glyphicon glyphicon-remove"></span>
```
Comment (mindreeper2420) [inline src/app/space-wizard/components/app-generator-single-selection-dropdown/app-generator-filtered-dropdown.component.html]: please change to `<span class="pficon pficon-close"></span>` to use the PatternFly icon. Thanks!

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1502-fabric8-ui.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1502-fabric8-ui.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1502-fabric8-ui.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1502-fabric8-ui.int.rdu2c.fabric8.io

**Comment by joshuawilson** [comment]
@mitchharpur can you please review this and answer @corinnekrych questions.

**Comment by corinnekrych** [comment]
closes this PR as issues is closed: https://github.com/fabric8io/fabric8-ui/issues/1489
the multi-selectect dropdown already have the filter.

**Pull request closed without merging**